### ISSUE TITLE
Chunks are not built until they become visible

### ISSUE THREAD

**Issue by Yadokingu** [jellysquid3/Sodium#118]
See video: https://youtu.be/9jKh74elJQM
Seems to happen with beacons that are under the bedrock in general, when you are above it.

![2020-07-20_00 33 02](https://user-images.githubusercontent.com/11133215/87886913-f23bf000-ca20-11ea-8e7b-268e148b39f1.png)

**Comment by jellysquid3** [comment]
Does this issue occur in vanilla as well? The game shouldn't build or load chunks that can't be seen, so the chunks which the beacons at the bottom of the world are located in will never load, and as such, the client will never render them.

**Comment by Yadokingu** [comment]
You are right. It is the same behaviour as in vanilla. Guess I played in optifine for too long.
It's just weird incosistent behaviour. Some beacons at bedrock level (the one at level 1) will activate, but this beacon at lava level, shown in the video, will only activate if you go below the upper bedrock. But again, it is the same inconsistent as with vanilla, so I guess it's fine.
If it is an easy change, it would be a great enhancement though, to always show beacon beams.

**Comment by jellysquid3** [comment]
I'm going to convert this bug report into a feature request, then. It's probably not out of the question for Sodium to spend some CPU time in the background building chunks you can't see, as they will eventually become visible anyways.
